Resources/Locale/en-US/connection-messages.ftl

```diff
@@ -22,6 +22,11 @@ ban-banned-1 = You, or another user of this computer or connection, are banned f
 ban-banned-2 = The ban reason is: "{$reason}"
 ban-banned-3 = Attempts to circumvent this ban such as creating a new account will be logged.
 
+restrict-username-1 = Your username is prohibited either specifically or by class.
+restrict-username-2 = The reason is: {$reason}
+restrict-username-3 = You may be welcome back on the server after your name is changed.
```

Comment:
Isn't this not always correct given that we have "extend to ban" bans here?

Reply:
bans are checked before username restrictions in the login manager, if a user is banned then the user will not see this message.

Reply:
Will the ban message take priority if the user gets on-the-spot banned in this case?